### PR TITLE
Add more flag icons for regional buckets

### PR DESCRIPTION
**Pull request by wnojopra**
This pull requests gets all the regions from https://cloud.google.com/storage/docs/locations and adds an appropriate flag icon for it. 

Example screenshot from local testing:
Screenshot 2021-03-10 4 15 06 PM

### CODE REVIEW

**Review by zarsky-broad — APPROVED** [review]
Looks fine to me. We're working on finding a second person to review.

File: src/components/region-common.js [hunk 1]
```diff
@@ -17,8 +17,48 @@ export const regionInfo = (location, locationType) => {
...
+        case 'SOUTHAMERICA-EAST1':
+          return { flag: '🇧🇷', regionDescription: `${locationType}: ${location} (Sao Paulo)` }
+        case 'US-CENTRAL1':
+        case 'US-EAST1':
+        case 'US-EAST4':
+        case 'US-WEST1':
+        case 'US-WEST2':
+        case 'US-WEST3':
+        case 'US-WEST4':
+          return { flag: '🇺🇸', regionDescription: `${locationType}: ${location}` }
```
Comment (breilly2) [inline src/components/region-common.js]: Should this show a city/country/whatever like the others do?

**Review by breilly2 — APPROVED** [review]
👍   Looks good to me. I didn't test (not exactly sure how, and there are a lot of possible cases), but I don't see any problems, and risk of seriously breaking anything is very low.

File: src/components/region-common.js (hunk 1 above)
Comment (wnojopra) [reply src/components/region-common.js]: Looked at this a little bit more, and I think this is fine as is. For non-US regions its sometimes important to know the city/country for regulatory reasons. Its less important for US regions, and showing the region name (e.g. US-WEST4) should suffice.

Comment (wnojopra) [reply src/components/region-common.js]: Actually now that I've worked on https://github.com/DataBiosphere/terra-ui/pull/2420/ I think we need the US cities too. I've added that.

**Comment by breilly2** [comment]
Still looks good to me. Are you ready to merge now or do you want to see if anything else falls out of #2420 ?

**Comment by wnojopra** [comment]
> Still looks good to me. Are you ready to merge now or do you want to see if anything else falls out of #2420 ?

This is good to merge. Would you mind merging this PR for me? I don't have write permission on this repo.

**Comment by breilly2** [comment]
Since this PR is not able to run the integration tests automatically in CircleCI, I ran them locally against this branch. They passed.

**Pull request merged**